svelte/components/Nft/NftTransfer.svelte

```diff
@@ -31,6 +39,9 @@
 
       transfered = Boolean(txReceipt.status);
       transfering = false;
+
+      nftStore.nftRemoveOne(chainId, address, tokenID, $metamaskAccount);
```

Comment:
an NFTRefresh on NFT component should be sufficient ? no need for a remove function ?